Lib9c/Action/Sweep.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Battle;
+using Nekoyume.Extensions;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionType("sweep")]
+    public class Sweep : GameAction
```

Comment:
If the 'Sweep' action only works without the worldId of mimisbrunnr, how about renaming it something like 'WorldSweep'?
The word 'World' seemed appropriate to describe something other than 'Mimisbrunnr'. Although `WorldSheet` contains `Mimisbrunnr` information.
Considering connectivity, you could also consider `HackAndSlashSweep`.